frontend/package.json

```
@@ -10,24 +10,29 @@
     "lint:css": "npx stylelint \"src/**/*.scss\"",
     "lint": "yarn lint:css && yarn lint:js",
     "start:dev": "vite",
-    "start:dev:local": "dotenv -e ../.env/frontend.env -- vite",
+    "start:dev:local": "dotenv -e ../.env/frontend.env -- vite --force",
```

The `--force` can be removed now
```suggestion
    "start:dev:local": "dotenv -e ../.env/frontend.env -- vite",
```